environment.rb

```diff
@@ -0,0 +1,102 @@
+module Environment
+
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: {
+          intelligence: %i[cellphone],
+          items: %i[medipack chevy_versa]
+        },
+        standard_transportation_pack: {
+          intelligence: %i[cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemestry chevy_versa]
+        },
+        simple_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemistry financial]
+        },
+        standard_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: %i[remington_870 colt_1911 machete hatchet],
+          items: %i[handcuffs medipack chemistry financial]
+        }
+      }
+    end
+
```

Comment:
This kind of extra lines can be taken by the [Rubocop](https://docs.rubocop.org/rubocop/1.23/index.html) gem as an offense, we should avoid them.